cost/volumes/uav_policy.pt

```diff
@@ -0,0 +1,288 @@
+﻿# TO-DO: 
+#	Add support for deleting old unattached volumes.
+#
+
+name "Unattached Volumes"
+rs_pt_ver 20180301
+short_description "Finds unattached volumes older than specified number of days and reports them."
+severity "medium"
+category "Cost"
+
+permission "general_permissions" do
+  resources "rs_cm.volumes", "rs_cm.audit_entries", "rs_cm.placement_groups", "rs_cm.clouds", "rs_cm.tags", "rs_cm.deployments"
+  actions   "rs_cm.*"
+end
+
+parameter "param_email" do
+  type "string"
+  label "Email address to send escalation emails to"
+  allowed_pattern /[a-zA-Z0-9_\-\.]+@[a-zA-Z0-9_\-\.]/
+  min_length 1
+  constraint_description "Must be an email address."
+end
+
+parameter "param_days_old" do
+  type "number"
+  label "Identify volumes that have been unattached for the given number of days."
+  default 30
+end
+
+parameter "param_delete" do
+  type "string"
+  label "Enter the word, DELETE (all UPPERCASE), to have identified volumes automatically deleted."
+  allowed_values "ALERT ONLY", "ALERT AND DELETE"
+  default "ALERT ONLY"
+end
+
+resources "clouds", type: "rs_cm.clouds" do
+  filter do
+    cloud_type ne: ["soft_layer", "cloud_stack", "rackspace_next_gen", "blue_skies","open_stack_v2","uca","open_stack_v3"]
+  end
+end
+
+datasource "ds_clouds" do
+	iterate @clouds
+	field "href", href(iter_item)
+	field "cloud_type", val(iter_item, "cloud_type")
+  field "name", val(iter_item,'name')
+end
+
+resources "pgs", type: "rs_cm.placement_groups" do
+	view "extended"
+end
+
+datasource "ds_pgs" do
+	iterate @pgs
+	field "id", val(iter_item, "resource_uid")
+	field "name", val(iter_item, "name")
+	field "account_type", val(val(iter_item, "cloud_specific_attributes"), "account_type")
+end
+
+resources "volumes", type: "rs_cm.volumes" do
+  iterate @clouds
+  cloud_href href(iter_item) 
+  view "extended"
+end
+
+# Processing volumes in javascript since extended view does not always have a placement_group or volume_type hash in the
```

Comment:
should this section be removed since it's not used.

Reply:
Yes. Removed.